core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java

```diff
@@ -184,6 +193,9 @@ public static class Builder
         private long myBackoffInMs = DEFAULT_BACKOFF_IN_MS;
         private boolean myIgnoreTWCSTables = DEFAULT_IGNORE_TWCS_TABLES;
         private TimeUnit myPriorityGranularityUnit = TimeUnit.HOURS;
+        private static final long DEFAULT_NEW_TABLE_SLIDEOFF = TimeUnit.DAYS.toMillis(1);
```

Comment:
Refactor constant name. There is no "new table slideoff".

Also, here the default value is one day, but you have set the value to 1 hour in the ecc.yml file.